tools/fvm-bench/src/fevm.rs

```diff
@@ -79,9 +97,147 @@ pub fn run(
 
     let invoke_res = testkit::fevm::invoke_contract(tester, &mut account, actor, &input_data, gas)?;
     let BytesDe(returnval) = invoke_res.msg_receipt.return_data.deserialize().unwrap();
+    println!("Revert Reason: {}", parse_eth_revert(&returnval).unwrap());
```

Comment:
We need to check the `exit_code` here. If it's 33, it's a revert. Otherwise, attempting to parse this as a revert will fail.

Reply:
I'd also call this to handle reverts from the constructor above (line 87).

Reply:
> We need to check the `exit_code` here. If it's 33, it's a revert. Otherwise, attempting to parse this as a revert will fail.

Sorry, I don't understand. The panic and revert errors also revert with exit code 33 only ig.

Reply:
Both done @Stebalien 🙂🚀.
I have included the checking in handleResult now, it would cover all cases imo.